assets/js/main.js

```diff
@@ -0,0 +1,167 @@
+(function() {
+  // ================== globals ==================
+  var COLORS = ['red', 'green', 'blue', 'yellow'];
+  var START_DIFFICULTY = 3;
+  var COLOR_LIGHT_TIME = 300;
+  var STARTING_TIME_PER_COLOR = 1000;
+
+  var sounds = {};
+
+  var $colors = {};
+  var $simon = null;
+  var $score = null;
+
+  var game = null;
+
+  // ================== utility ==================
+  // Returns a random integer between min (included) and max (excluded)
+  // Using Math.round() will give you a non-uniform distribution!
+  var getRandomInt = function(min, max) {
```

Comment:
I opted to use the `var functionName = function() {}` instead of `function functionName() {}` any thoughts?  I think we should probably pick one and stick with it whichever way we go.  Khan academy uses `var func = `